### PR TITLE
VN-370 Add pay_me_now section

### PR DESCRIPTION
**Pull request by huy2nhan**
- Add section `pay_me_now` English
- Add translation for `vi` of `pay_me_now`

### CODE REVIEW

File: en/index.json [hunk 1]
```diff
@@ -112,5 +112,12 @@
     "electric_cost": "Electricity cost for 1 kWh",
     "estimate_again": "Estimate again",
     "use_detected_wattage" : "Use wattage data obtained from Miner"
+  },
+  "pay_me_now": {
+    "pay_me_now": "Pay Me Now!",
+    "payment_queued": "Payment queued.",
+    "payment_verification": "Payment Verification",
+    "payment_verification_explain": "To verify if your account can receive payments from Freedom!, we will pay this account an amount of $0.01",
+    "verify": "VERIFY"
```
Comment (pandafox) [inline en/index.json]: Lower case?

Comment (pandafox) [reply en/index.json]:
```
“pay_me_now”：“馬上付錢！”，
    “payment_queued”：“已申請支付”，
    “payment_verification”：“付款驗證”，
    “payment_verification_explain”：“要驗證您的帳戶是否可以接受Freedom！的付款，我們將向此帳戶支付0.01美元”，
    “verify”：“驗證”,
    “verified”：“已驗證”
```

Is verify showing already verified badge? If so, the text should be "Verified" instead?

**Review by pandafox — CHANGES_REQUESTED** [review]

File: en/index.json (hunk 1 above)
Comment (huy2nhan) [reply en/index.json]: image
it is for caption of button. I'm not so sure uppercase is different in other language or not (like Chinese) so I put it uppercase. For now, I guess we can do upper case in code.

Comment (pandafox) [reply en/index.json]: I see, in that case I think the original looks fine.

Comment (huy2nhan) [reply en/index.json]: I added Chinese translation.

**Review by pandafox — DISMISSED** [review]

**Comment by pandafox** [comment]
@angels2it would this work for other language if they dont' have the same number of keys? (ex: fallback to english?)

**Comment by angels2it** [comment]
Yes, It will fallback to English if keys does not exist. We usually copy default keys from English value to other language for make sure we don't miss this key when translate.

**Review by votuanphong1983 — DISMISSED** [review]

**Review by votuanphong1983 — APPROVED** [review]

**Pull request merged**